client/src/components/Modal/HeaderModal.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "styled-components";
+
+const ModalHead = styled.header`
```

Comment:
naming it like the html sibling is maybe easier to read later, especially because you didn't include logic, just styling
```suggestion
const Header = styled.header`
```

Reply:
Thanks for suggestion! 😊👍🏻